ui/src/app/pages/admin/admin-notebook-view.tsx

```diff
@@ -97,7 +98,11 @@ const AdminNotebookViewComponent = (props: Props) => {
   </React.Fragment>;
 };
 
-const AdminNotebookView = () => {
+const AdminNotebookView = (spinnerProps: WithSpinnerOverlayProps) => {
+  useEffect(() => {
+    spinnerProps.hideSpinner();
+  }, [spinnerProps.spinnerVisible]);
```

Comment:
is it not sufficient to just use `[]` as the useEffect dependency? I thought that was basically the equivalent of `componentDidMount`

Reply:
Looks like it does work, thanks for the tip!